tests/playwright-tests/utils/generateRunIdentifier.ts

```diff
@@ -0,0 +1,5 @@
+export default function generateRunIdentifier() {
+  const date = new Date();
+  const utcFormat = date.toISOString().replace('T', ' ').substring(0, 19);
+  return `UI Test${utcFormat}`;
```

Comment:
For the run identifier, we normally append an additional random string that's 6 characters in length. 

This provides some uniqueness to prevent collisions with other test runs that are running in parallel. This avoids multiple tests run all having the same run identifier.

Would suggest we do something similar here.

The best way to do this would be to have some util like:

```ts
export default function generateRandom(length: number): string {
  let random = '';

  // Avoid empty string being created if `Math.random` creates 0
  while (!current) {
    random = Math.random().toString(36).slice(2, 2 + length);
  }

  return random;
}
```

This would then be appended to the run identifier string i.e.

```ts
export default function generateRunIdentifier(): string {
  // ...
  return `${utcFormat} ${generateRandom(6)}`
}
```